ibis/omniscidb/client.py

```diff
@@ -470,6 +470,38 @@ def _run_ddl(**kwds):
             _run_ddl, tbl_properties=tbl_properties
         )
 
+    def insert_into(self, dst_values, dst_cols=None):
```

Comment:
@YarShev  `ibis.omniscidb` already has a method for loading data: https://github.com/ibis-project/ibis/blob/e60d1afd3648bb8abefdf81ff040acc4cf3a5698/ibis/omniscidb/client.py#L1069

why `insert_into` method here is different than `load_data` ?

Reply:
Can say when `load_data` takes `obj` as iterable of tuples then it performs the same action as `insert_into`. But, I think `insert_into` has some pros:
1) If we interact with `OmniSciDBTable` object then we can continue interacting with that object, but not apply to methods of `OmniSciDBClient`.
2) `insert_into` method is more SQL like
3) It's possible not specify columns in which values will be inserted.
What do you think?